src/components/MatchStats/MatchStatsTeamTable.tsx

```diff
@@ -156,22 +165,18 @@ const MatchStatsTeamTable: React.FC<{ teamName: string; teamData: any[], display
           {rows.map((row: any) => {
             prepareRow(row);
             return (
-              <Tr {...row.getRowProps()}>
+              <Tr
+                {...row.getRowProps()}
+                className={styles.tableRow}
+                onClick={createGoToPlayerPage(row.original.playerId)}
```

Comment:
Can you please add functionality that says 
if playerid starts with 8e6a51baf1c7e338a118d9e32472954 or 58e419de5a8b2655f6d48eab68275db - do nothing
?

These are shared guids and long story short links will not work